src/ProgramBlockEditor.js

```diff
@@ -177,35 +180,52 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, {}> {
+                <Row className='ProgramBlockEditor__header'>
+                    <Col className='ProgramBlockEditor__title'>
+                        <FormattedMessage id='ProgramBlockEditor.programLabel' />
+                    </Col>
+                    <div className='ProgramBlockEditor__editor-actions'>
                         <Button
                             key='addButton'
-                            className='ProgramBlockEditor__editor-action-button'
+                            className={this.addIsSelected() ?
+                                        'ProgramBlockEditor__editor-action-button ProgramBlockEditor__editor-action-button--pressed' :
+                                        'ProgramBlockEditor__editor-action-button'}
                             aria-pressed={this.addIsSelected() ? 'true' : 'false'}
                             aria-label={this.props.intl.formatMessage({id:'ProgramBlockEditor.editorAction.add'})}
-                            variant={this.addIsSelected() ? 'outline-primary' : 'light'}
                             onClick={this.handleClickAdd}>
-                            <AddIcon/>
+                            <AddIcon className='ProgramBlockEditor__editor-action-button-svg'/>
                         </Button>
                         <Button
                             key='deleteButton'
-                            className='ProgramBlockEditor__editor-action-button'
+                            className={this.deleteIsSelected() ?
+                                        'ProgramBlockEditor__editor-action-button ProgramBlockEditor__editor-action-button--pressed' :
+                                        'ProgramBlockEditor__editor-action-button'}
                             aria-pressed={this.deleteIsSelected() ? 'true' : 'false'}
                             aria-label={this.props.intl.formatMessage({id:'ProgramBlockEditor.editorAction.delete'})}
-                            variant={this.deleteIsSelected() ? 'outline-primary' : 'light'}
                             onClick={this.handleClickDelete}>
-                            <DeleteIcon/>
+                            <DeleteIcon className='ProgramBlockEditor__editor-action-button-svg'/>
                         </Button>
-                    </Col>
+                    </div>
                 </Row>
                 <Row>
-                    <Col>
+                    <Col className='ProgramBlockEditor__program-sequence'>
                         {programBlocks}
                     </Col>
                 </Row>
-            </div>
+                <Row className='ProgramBlockEditor__footer'>
+                    <Col>
+                        <Button
+                            aria-label={`${this.props.intl.formatMessage({id:'PlayButton.run'})} ${this.props.program.join(' ')}`}
+                            className='ProgramBlockEditor__run-button'
+                            disabled={this.props.runButtonDisabled !== 'connected'}
```

Comment:
Please update to:

    disabled={this.props.runButtonDisabled}

Reply:
oops!